velox/exec/TableWriter.cpp

```diff
@@ -55,7 +58,11 @@ TableWriter::TableWriter(
   const auto& connectorId = tableWriteNode->insertTableHandle()->connectorId();
   connector_ = connector::getConnector(connectorId);
   connectorQueryCtx_ = operatorCtx_->createConnectorQueryCtx(
-      connectorId, planNodeId(), connectorPool_);
+      connectorId,
+      planNodeId(),
+      connectorPool_,
+      spillConfig_.has_value() ? &(spillConfig_.value()) : nullptr,
+      &numSpillRuns_);
```

Comment:
I think we can't pass numSpillRuns_ from operator to sort writer as it needs concurrency control. Let SortingWriter object to pass its own numSpillRuns to SortingWriter as for now.